test/screenshot/webpack-bundles.js

```diff
@@ -21,6 +21,14 @@ module.exports.bundle = function(testPath, outputPath) {
             {
               loader: 'css-loader',
             },
+            {
```

Comment:
screenshot tests looked strange without this...I noticed that the `appearance: none` rule was not being applied. Chrome needs the `-webkit-appearance: none` rule to apply.